sandbox

```diff
@@ -260,11 +265,11 @@ sandbox () {
 
   clean () {
     echo "* docker-compose down"
-    docker-compose down -t 0                                   || true
+    dc down -t 0                                   || true
     echo "* docker rmi sandbox_algod sandbox_indexer"
     docker rmi sandbox_algod sandbox_indexer                   || true
```

Comment:
Same question with docker, I think these might be even more CWD dependent, but maybe not the "rmi" subcommand...

Reply:
You might be thinking of the fact that docker, by default, copies the current directory (and children) into the image.  But we're only building images with docker-compose so I don't think that'd be an issue.  If it were, I would have seen it when I first did an "up" while I was outside the sandbox directory.  But that was fine.